mapfile.c

```diff
@@ -3991,7 +3991,7 @@ int loadLayer(layerObj *layer, mapObj *map)
         if(getString(&layer->connection) == MS_FAILURE) return(-1); /* getString() cleans up previously allocated string */
         break;
       case(CONNECTIONTYPE):
-        if((type = getSymbol(12, MS_OGR, MS_POSTGIS, MS_WMS, MS_ORACLESPATIAL, MS_WFS, MS_GRATICULE, MS_PLUGIN, MS_UNION, MS_UVRASTER, MS_CONTOUR, MS_KERNELDENSITY, MS_IDW)) == -1) return(-1);
+        if((type = getSymbol(12, MS_OGR, MS_POSTGIS, MS_WMS, MS_ORACLESPATIAL, MS_WFS, MS_GRATICULE, MS_PLUGIN, MS_UNION, MS_UVRASTER, MS_CONTOUR, MS_KERNELDENSITY, MS_IDW, MS_FLATGEOBUF)) == -1) return(-1);
         layer->connectiontype = type;
```

Comment:
I wonder what is the meaning of the number in `getSymbol(12`.  Could it mean the number of items to follow and thus it should be increased by one?

Reply:
Ah! indeed! I was scratching my head over why I still got parsing error... now I get Unknown connectiontype, it was 19 to be expected so I can begin work on implementing my connectiontype. Thanks @jratike80!